husky_components/datagrid/decorators/tiles-view.js

```diff
@@ -166,16 +166,23 @@ define(function() {
         renderRecord: function(record) {
             record = processContentFilters.call(this, record);
             var title = this.getTitleString(record),
-                description = this.getDescriptionSring(record);
+                description = this.getDescriptionSring(record),
+                lineHeight, countLinesOfTitle;
+
             this.$tiles[record.id] = $(_.template(templates.tile, {
                 icon: this.options.icon,
                 title: title,
-                croppedTitle: this.sandbox.util.cropMiddle(this.getTitleString(record), 20),
-                description: this.sandbox.util.cropMiddle(description, 20)
+                description: this.sandbox.util.cropMiddle(description, 18)
             }));
             this.$tiles[record.id].data('id', record.id);
 
             this.$el.append(this.$tiles[record.id]);
+
+            lineHeight = parseInt(this.$tiles[record.id].find('.title').css('line-height').replace('px', ''));
+            countLinesOfTitle = this.$tiles[record.id].find('.title').height() / lineHeight;
+            if (countLinesOfTitle > 2) {
+                this.$tiles[record.id].find('.title').html(this.sandbox.util.cropMiddle(title, 18));
```

Comment:
What's happening here? Haven't the title already be cropped with these parameters a few lines before?